### PR TITLE
Websockets refactor

### LINKED ISSUE #53: Warning: Too many active WebGL contexts
After switching between different pages and the Digital Twin/Viewer page, this error appears:
```viewer.ts:95 WARNING: Too many active WebGL contexts. Oldest context will be lost.```

Seems like the `WebGLRenderer` is not disposed correctly after we leave the page.

### PR DESCRIPTION
**Pull request by grzpiotrowski**
This supposed to be just general enhancements update but evolved into quite a refactor of handling Websocket connections.
Something that I wanted to do for quite a while now but was a real rabbit hole.

This work comes from an issue that my Viewer page was initialising the Websocket connections to the Kafka topics listed in the projects.
This was all fine until we navigated around the application, and since it is an SPA application, the Websocket connections kept living and sending/receiving data. Then we came back to the page to initialise more WS connections on top of the already existing ones. If the `clientId` was the same, then the connection was not even allowed and the app errored. This is why at some point as a hacky solution I introduced the `clientId` generated with the current timestamp.
But this meant that every time we came back to the page, we opened a new set of websocket connections and received all the data from Kafka again.

There was a need for a centralised websocket manager. Web Worker would probably be ideal, especially a shared one, but after a bit of trying I think I have to drop this idea and because of that the Websockets are now kept in a Svelte store. Still initialised in the Viewer page, but then when we come back to it it checks for existing connections and only creates new listeners (which it clears `onDestroy` when we leave the page.

This allowed to also use the existing websocket connections on other pages, by creating new listeners on them.

In short in this PR we have:
- Live sensor data graph on each Sensor details page.
- Latest readings for each sensor in a room on Room details page.
- Cleaned up Projects listing page.
- Sensors glowing for 500ms when they receive data, because it's nice to look at.
- Added a simple landing page.
- Fixes #53 by disposing of the renderer in Viewer page `onDestroy`

### CODE REVIEW

**Review by grzpiotrowski — COMMENTED** [review]
🏁
